torch/_inductor/virtualized.py

```diff
@@ -95,6 +95,11 @@ class NullHandler:
     """
 
 
+# If a virtualized value is set to _PoisonedVirtual then any attempt to get the
+# value will result an an exception being raised.
+_PoisonedVirtual = object()
```

Comment:
Let's add a bit more to the comment to explain when poisoning a virtualized is useful